code/email/MailblockMailSubscriber.php

```diff
@@ -94,29 +99,5 @@ public function beforeSendPerformed(\Swift_Events_SendEvent $evt)
             $message->setBcc($bcc);
             $message->setCc($cc);
         }
-        else {
```

Comment:
Everything from this else statement down is just double handling stuff that core already does